koku/masu/processor/tasks.py

```diff
@@ -293,7 +294,7 @@ def remove_expired_data(schema_name, provider, simulate, provider_uuid=None, que
 
 
 @celery_app.task(name="masu.processor.tasks.summarize_reports", queue=SUMMARIZE_REPORTS_QUEUE)
-def summarize_reports(reports_to_summarize, queue_name=None, manifest_list=None):
+def summarize_reports(reports_to_summarize, queue_name=None, manifest_list=None):  # noqa C901
```

Comment:
Rather than accepting the increased complexity, try and make this simpler. It's right on the edge at 10 right now.

Reply:
@samdoran I truly appreciate this comment and suggestion below.